bundles/org.eclipse.swt/Eclipse SWT Browser/common/org/eclipse/swt/browser/Browser.java

```diff
@@ -1187,6 +1189,17 @@ public boolean setUrl (String url, String postData, String[] headers) {
 	return webBrowser.setUrl (url, postData, headers);
 }
 
+/**
+ * Checks if the location supplied is the location for setting custom text in the browser
+ *
+ * @param location the URL to be checked
+ *
+ * @since 3.127
+ */
+public boolean isLocationForCustomText(String location) {
+	return URI.create(location).equals(webBrowser.getBaseUri());
```

Comment:
I propose to have a null check for `location` here. Either specify what happen when passing null (such as always returning false for a null value), or explicitly specifying in the Javadoc that null is not allowed and additionally explicitly checking the parameter to have an early failure:
```suggestion
	return URI.create(Objects.requireNonNull(location)).equals(webBrowser.getBaseUri());
```

Reply:
Done

Reply:
Nothing changed in the specification, only the null check was added to the code.

Reply:
Added in the parameter specification